src/renderer/ui/components/settings/ToggleableOption.js

```diff
@@ -9,6 +9,11 @@ class SettingsCheckbox extends Component {
     hintLabel: PropTypes.string,
     settingsKey: PropTypes.string.isRequired,
     setSetting: PropTypes.func.isRequired,
+    muiTheme: PropTypes.object,
+  };
+
+  static contextTypes = {
```

Comment:
Should really use the helper from MUI

https://github.com/callemall/material-ui/blob/master/src/styles/muiThemeable.js